extensions/mssql/notebooks/TSG/cluster-status.ipynb

```diff
@@ -0,0 +1,167 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": "# Cluster Status Notebook\nThis notebook allows you to see the status of the controller, master instance, and pools in your SQL Server big data cluster.",
+            "metadata": {}
+        },
+        {
+            "cell_type": "code",
+            "source": "import sys, platform\r\n\r\nif platform.system()==\"Windows\":\r\n    user = ' --user'\r\nelse:\r\n    user = ''\r\n    \r\ncmd = f'{sys.executable} -m pip uninstall --yes mssqlctl-cli-storage'\r\ncmdOutput = !{cmd}\r\ncmd = f'{sys.executable} -m pip uninstall -r http://helsinki/browse/packages/python/aris-p-release-candidate-gb/mssqlctl/requirements.txt --yes'\r\n\r\ncmdOutput = !{cmd}\r\ncmdOutput = ''.join(cmdOutput)\r\nif 'is not installed' in cmdOutput or 'Successfully uninstalled mssqlctl' in cmdOutput:\r\n    print(\"Uninstalling mssqlctl successful: \" + cmd)\r\nelse:\r\n    raise SystemExit(f'Uninstall of mssqlctl failed:\\n\\n\\t{cmd}\\n\\nreturned non-zero exit code: ' + ''.join(cmdOutput) + '.\\n')\r\n\r\ncmd = f'{sys.executable} -m pip install -r http://helsinki/browse/packages/python/aris-p-release-candidate-gb/mssqlctl/requirements.txt{user} --trusted-host helsinki'\r\nprint(\"Installing the latest version of mssqlctl: \" + cmd)\r\ncmdOutput = !{cmd}\r\ncmdOutput = ''.join(cmdOutput)\r\nif 'Requirement already satisfied' in cmdOutput or 'Successfully installed mssqlctl' in cmdOutput:\r\n    print(f'\\nSUCCESS: Upgraded the mssqlctl to the latest version')\r\nelse:\r\n    raise SystemExit(f'Installation of mssqlctl failed:\\n\\n\\t{cmd}\\n\\nreturned non-zero exit code: ' + ''.join(cmdOutput) + '.\\n')\r\n\r\n#install pandas\r\ncmd = f'{sys.executable} -m pip show pandas'\r\ncmdOutput = !{cmd}\r\nif len(cmdOutput) > 0 and '0.24' in cmdOutput[1]:\r\n    print('Pandas required version is already installed!')\r\nelse:\r\n    pandasVersion = 'pandas==0.24.2'\r\n    cmd = f'{sys.executable} -m pip install {pandasVersion}'\r\n    cmdOutput = !{cmd}\r\n    print(f'\\nSuccess: Upgraded pandas.')\r\n",
+            "metadata": {},
+            "outputs": [
```

Comment:
Outputs still need to be cleared

Reply:
I'll make sure to clear the output on the final copy that Alan Yu sends me with documentation.